unix/vncpasswd/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ add_executable(vncpasswd
   vncpasswd.cxx)
 
 target_include_directories(vncpasswd PUBLIC ${CMAKE_SOURCE_DIR}/common)
-target_link_libraries(vncpasswd tx rfb os)
+target_link_libraries(vncpasswd tx rfb os pwquality)
```

Comment:
How commont is pwquality? Is it available for all distributions? Otherwise it will need to be an optional dependency.

Could you please also add it to BUILDING.txt?

Reply:
I checked and tested the pwqaulity.pc of the rpm and deb based distribution (centos/debian)
The compilation link parameters are all "- l pwquality"
It should be universal, and I will test and check other distributions later.

Reply:
Compiling on Debian requires installing the libpwquality-dev package.